lms/djangoapps/course_api/blocks/api.py

```diff
@@ -94,6 +104,8 @@ def get_blocks(
     # TODO: Remove this after REVE-52 lands and old-mobile-app traffic falls to < 5% of mobile traffic
     if is_request_from_mobile_app(request):
         transformers += [HideEmptyTransformer()]
+        if ENABLE_VIDEO_URL_REWRITE.is_enabled(usage_key.course_key):
+            transformers += [VideoBlockURLTransformer()]
```

Comment:
Rather than putting this code here, can you add this transformer to `https://github.com/edx/edx-platform/blob/e7a509faf0e5f45b23457c69ddd3c3652780690d/lms/djangoapps/course_api/blocks/transformers/blocks_api.py`?

This transformer is specific to what is returned via the Blocks API so it makes sense to have it be a "sub-transformer" of the `BlocksAPITransformer`.

Reply:
The rewrite is mobile-exclusive. I have added a detailed comment below why it is so.